torchbenchmark/models/pplbench_beanmachine/__init__.py

```diff
@@ -0,0 +1,76 @@
+import time
+import random
+import torch
+
+from ...util.model import BenchmarkModel
+from torchbenchmark.tasks import OTHER
+from typing import Tuple
+
+from pplbench.ppls.beanmachine import robust_regression
+from pplbench.models.robust_regression import RobustRegression
+from pplbench.ppls.beanmachine.inference import MCMC
+
+class Pplbench(torch.nn.Module):
+    def __init__(self):
+        super(Pplbench, self).__init__()
+
+        # Instantiate model
+        self.model = RobustRegression()
+
+        # Get data for evaluating model
+        self.train_data, self.test_data = self.model.generate_data(seed=int(time.time()))
+
+        # Create inference object with training data
+        self.infer_obj = MCMC(robust_regression.RobustRegression,
+                              self.train_data.attrs)
+        self.infer_obj.compile()
+
+    def forward(self, train_data, test_data, training=False):
+
+        if training:
+            # Run bayesian inference (training) on given data
+            samples = self.infer_obj.infer(data=train_data, iterations=1500, num_warmup=0, seed=random.randint(1, int(1e7)))
+
+        if not training:
+            # Run bayesian inference (training) on given data for 1 iteration
+            # We need the object of type MonteCarloSamples for the evaluation step
+            # @Todo Can we create samples object without using infer function?
+            samples = self.infer_obj.infer(data=train_data, iterations=1, num_warmup=0, seed=random.randint(1, int(1e7)))
```

Comment:
Can we increase the number of iterations here for inference, to increase the computation workload? Otherwise it may finishes too quick, and the performance signals will be shadowed by noises.